<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizationerClass</class>
 <widget class="QMainWindow" name="VisualizationerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualizationer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="ControlPanel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="layoutStrategyLabel">
           <property name="text">
            <string>LayoutStrategy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="layoutComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="degreeSearchLabel">
           <property name="text">
            <string>SearchByDegree</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="searchSpinBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="findButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="searchLabel">
           <property name="text">
            <string>SearhNextLayer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchPushButton">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="flitterLabel">
           <property name="text">
            <string>Flitter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="fillterDial"/>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="showSelectionlabel">
           <property name="text">
            <string>ShowSelections</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="focusPushButton">
           <property name="text">
            <string>Focus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="showAllLabel">
           <property name="text">
            <string>ShowAll</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="defocusButton">
           <property name="text">
            <string>DeFocus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="showNodesBox">
           <property name="text">
            <string>ShowNodes</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="showEdgesBox">
           <property name="text">
            <string>ShowEdges</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="dragModeGroupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>SelectMode</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QRadioButton" name="drugRadioButton">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>43</y>
            <width>47</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Drag</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="selectRadioButton">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>21</y>
            <width>59</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Select</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="colorGroupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>ColorMode</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QRadioButton" name="singleRadioButton">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>89</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>SingleColor</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QRadioButton" name="multiRadioButton">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>83</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>MultiColor</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget" native="true">
          <property name="geometry">
           <rect>
            <x>-10</x>
            <y>-240</y>
            <width>10</width>
            <height>524</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="DataViewer" name="dataViewer"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="edgeDetailLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>EdgeDetail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="edgeTextBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nodeDetailLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>NodeDetail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="nodeTextBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoadData">
    <property name="title">
     <string>LoadData</string>
    </property>
    <addaction name="actionPaperConferenceAuthor"/>
    <addaction name="actionTopic"/>
   </widget>
   <addaction name="menuLoadData"/>
  </widget>
  <action name="actionPaperConferenceAuthor">
   <property name="text">
    <string>PaperConferenceAuthor</string>
   </property>
  </action>
  <action name="actionTopic">
   <property name="text">
    <string>Topic</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DataViewer</class>
   <extends>QGraphicsView</extends>
   <header>dataviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="visualizationer.qrc"/>
 </resources>
 <connections/>
</ui>
